multiqc/modules/hicexplorer/hicexplorer.py

```diff
@@ -0,0 +1,184 @@
+from multiqc.modules.base_module import BaseMultiqcModule
+import logging
+from collections import OrderedDict
+
+from multiqc import config
+from multiqc.plots import bargraph
+
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='HiCExplorer', anchor='hicexplorer',
+                                            href="https://hicexplorer.readthedocs.io",
+                                            info=" addresses the common tasks of Hi-C analysis from processing to visualization.")
+
+        self.mod_data = dict()
+        for file in self.find_log_files('hicexplorer'):
+            self.mod_data[file['s_name']] = self.parse_logs(file['f'])
+            self.add_data_source(file)
+        if len(self.mod_data) == 0:
+            raise UserWarning
+        self.colors = ["#1f77b4",
+                       "#ff7f0e",
+                       "#2ca02c",
+                       "#d62728",
+                       "#9467bd",
+                       "#8c564b",
+                       "#7f7f7f",
+                       "#bcbd22",
+                       "#17becf",
+                       "#D2691E"]
+        # prepare the basic statistics for hicexplorer
+        self.hicexplorer_basic_statistics()
+
+        # key lists for plotting
+        keys_list_plot_pairs = ['Pairs used', 'One mate unmapped', 'One mate not unique', 'One mate low quality',
+                                'Dangling end', 'One mate not close to rest site', 'Self circle', 'Duplicated pairs']
+        keys_list_contact_distance = ['Inter chromosomal', 'Short range', 'Long range']
+        keys_list_read_orientation = ['Inward pairs', 'Outward pairs', 'Left pairs', 'Right pairs']
+
+        # prepare the detail report section
+        self.add_section(
+            name='Pair categorization',
+            anchor='hicexplorer_pairs_categorized',
+            plot=self.hicexplorer_create_plot(keys_list_plot_pairs, 'HiCExplorer: Categorization of reads'),
+            helptext='''
+            This figure contains the number of reads that were finally used to build the 
```

Comment:
This is great, but now maybe a bit too far in the other direction! It's nice to have a short one-line description above the plot always showing (just not long paragraphs). So this first sentence would be perfect as a description, then the rests as `helptext`.